src/controllers/hid/hidcontroller.cpp

```diff
@@ -208,8 +220,22 @@ bool HidController::poll() {
             return false;
         } else if (bytesRead == 0) {
             // No packet was available to be read
+            for (int rptIdx = 0; rptIdx <= kMaxHidInputReportId; rptIdx++) {
+                if (inputReportCounter[rptIdx] > 1) {
+                    qCWarning(m_logOutput)
+                            << getName() << ": Read "
+                            << inputReportCounter[rptIdx]
+                            << " HID input reports with Report ID " << rptIdx
+                            << "in the same polling interval. This indicates a "
+                               "performance problem with controller mapping or "
+                               "a too slow polling interval.";
```

Comment:
How can we adjust the polling interval? If it is fixed, we can report the fixed time here.

Reply:
The polling intervall is defined here: https://github.com/mixxxdj/mixxx/blob/24c26c79796da4a86318c54dc4c65cfe2b2dd2f2/src/controllers/controllermanager.cpp#L28
I'm currently looking into optimizing this behavior. The correct setting for the HID polling interval is stored in the USB Endpoint Descriptor of the device, but this information is not provided via hidapi.

Reply:
This message makes the performance issue worse. Do you expect that this is only a development issue, or does it also happen during a gig? 
Even if Controller logging is disabled, this message will be assembled. 
Not sure how, but there must be a way to check if controller debugging is enabled and skip the whole feature.

Reply:
> I'm currently looking into optimizing this behavior.

Cool, Thank you. In the meanwhile I would just drop:
> "a too slow polling interval.";